File: input_program/program_10.java
import java.util.Scanner;

public class CharacterCounter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String text = scanner.nextLine();
        System.out.print("Enter the character to count: ");
        // Read the first character of the next input line
        char targetChar = scanner.nextLine().charAt(0);

        int count = 0;
        // Iterate through the string's characters
        for (int i = 0; i < text.length(); i++) {
            if (text.charAt(i) == targetChar) {
                count++;
            }
        }

        System.out.println("The character '" + targetChar + "' appears " + count + " times.");
        scanner.close();
    }
}

File: translated/gemini/program_10.py
def main():
    text = input("Enter a string: ")
    target_char = input("Enter the character to count: ")[0]

    count = 0
    for i in range(len(text)):
        if text[i] == target_char:
            count += 1

    print("The character '" + target_char + "' appears " + str(count) + " times.")

if __name__ == "__main__":
    main()